profiler/profiler.go

```diff
@@ -160,8 +160,7 @@ func (p *profiler) collect(ticker <-chan time.Time) {
 			for t := range p.cfg.types {
 				prof, err := p.runProfile(t)
 				if err != nil {
-					fmt.Printf("error: %v\n", err)
```

Comment:
@pmbauer I suspect you left this in 634b2317 by accident?

Reply:
looks like it

Reply:
@pmbauer ok, thanks for confirming, just wanted to make sure it wasn't intentional. I mean it seems like this is definitely an error path that'd be very unexpected for us to hit.